src/network/networkd-link.c

```diff
@@ -909,6 +914,7 @@ static int address_handler(sd_netlink *rtnl, sd_netlink_message *m, Link *link)
 
         if (link->address_messages == 0) {
                 log_link_debug(link, "Addresses set");
+                link->addresses_configured = true;
                 link_enter_set_routes(link);
         }
```

Comment:
it might be nice to eventually split out the five lines above into a new call `link_address_test_ready()` or so, which we can then call both from `address_handler()` and from `link_enter_set_addresses()`, since both execute kinda the same steps.

(But it's OK to leave this for now, and do that in another PR)